src/main/java/org/javarosa/core/model/instance/geojson/GeojsonGeometry.java

```diff
@@ -32,7 +32,7 @@ public class GeojsonGeometry {
     public String getOdkCoordinates() throws IOException {
         if (type.equals("Point")) {
             return coordinates.get(1) + " " + coordinates.get(0) + " 0 0";
-        } else if (type.equals("LineString")) {
+        } else if (type.equals("LineString") || type.equals("MultiPoint")) {
```

Comment:
It occurs to me that `MultiPoint` is semantically distinct from `LineString` and we probably shouldn't support it (yet). It's intended to be displayed without the lines in between so we don't have a way of representing that in the ODK geo notation.

Reply:
> It's intended to be displayed without the lines in between so we don't have a way of representing that in the ODK geo notation.

It was pretty unclear to me in the spec whether that was the case or not. I guess it's an array of points that can be empty or just one point so that implies no "line". Is the "0-dimensional Point and MultiPoint" comment what suggests that to you or is there another part that lays that out in a clearer way?

Either way, part of me feels like there wouldn't be a problem in showing it with a line now and enhancing it later? Maybe there are cases where the line would be problematic that I haven't thought about.

Reply:
Agreed the spec is not super clear but it was seeing the name again in context that made me realize the difference in intent. The Internet agrees: https://stackoverflow.com/questions/36685195/whats-the-difference-between-linestring-and-multipoint-in-geojson

> wouldn't be a problem in showing it with a line now and enhancing it later

It just doesn't make sense. Let's say the feature is "nurse stations" and there's a multipoint geometry. Drawing it with lines would not be useful I think it's more helpful to users to let them know we don't support what they're trying to do. In that case, they should disaggregate into one point per station.

That said, if you really feel strongly about keeping it in, this is not the hill I want to die on!

Reply:
> It just doesn't make sense. Let's say the feature is "nurse stations" and there's a multipoint geometry. Drawing it with lines would not be useful I think it's more helpful to users to let them know we don't support what they're trying to do. In that case, they should disaggregate into one point per station.

That makes a lot of sense. I'll remove the support here.